springdb/frontend/src/theme/typography.js

```diff
@@ -88,13 +88,13 @@ const typography = {
     lineHeight: 1.5,
     fontSize: pxToRem(12),
     letterSpacing: 1.1,
-    textTransform: 'uppercase'
+    textTransform: "uppercase"
   },
   button: {
     fontWeight: 700,
     lineHeight: 24 / 14,
     fontSize: pxToRem(14),
-    textTransform: 'capitalize'
+    textTransform: "capitalize"
```

Comment:
do you have a linter that automagically change these syntax things or something?